### PR TITLE
Use case edits for paper

### PR DESCRIPTION
**Pull request by kellynm**

### CODE REVIEW

**Comment by kellynm** [comment]
The purpose of this PR is to add/update notebooks used in the simulation manuscript. Major changes include:

- Add Jupyter notebook (paper_example_plots.ipynb) for plotting examples of contamination and sampling configurations, with two scenario configuration files used in notebook.
- Update use cases Jupyter notebook (simulation_use_cases.ipynb) and the three scenario configuration files used in the notebook to match the analysis used in paper. Add some description in notebook as markdown cells.
- Update validation notebook with correct markdown descriptions.

File: tests/test_summaries/inspection_scenarios.csv [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+inspection scenario 1,aqim calibrated,item hypergeometric random cluster,aqim_box_insp_unit.csv,"[0.0091, 3.3769]",item,clustered,random,40,40,items,hypergeometric,,0.05,cluster,0.1,random
+inspection scenario 2,aqim calibrated,item hypergeometric interval cluster,aqim_box_insp_unit.csv,"[0.0091, 3.3769]",item,clustered,random,40,40,items,hypergeometric,,0.05,cluster,0.1,interval
+inspection scenario 3,aqim calibrated,item hypergeometric convenience,aqim_box_insp_unit.csv,"[0.0091, 3.3769]",item,clustered,random,40,40,items,hypergeometric,,0.05,convenience,1,
+inspection scenario 4,aqim calibrated ,item hypergeometric random,aqim_box_insp_unit.csv,"[0.0091, 3.3769]",item,clustered,random,40,40,items,hypergeometric,,0.05,random,1,
```
Comment (wenzeslaus) [inline tests/test_summaries/inspection_scenarios.csv:5]:
```suggestion
inspection scenario 4,aqim calibrated,item hypergeometric random,aqim_box_insp_unit.csv,"[0.0091, 3.3769]",item,clustered,random,40,40,items,hypergeometric,,0.05,random,1,
```

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
I don't have anything major, but some final touches would be nice.

Missing "are" in "Note that the contamination rate distribution parameters estimated by running the simulation"? (in simulation_use_cases.ipynb)

`resultsdir = "H:/Shared drives/APHIS  P...`, `aqim_box_insp_unit.csv`, maybe others: Can you just remove `resultsdir` or make it `"."` (the current dir)? For `aqim_box_insp_unit.csv` you probably need to mention somewhere that it is not included in the repo and thus the notebook will not work. Given that, a computed notebook makes more sense here.

You can try adding the other two notebooks [into CI](https://github.com/ncsu-landscape-dynamics/popsborder/blob/main/.github/workflows/ci-notebooks.yml#L32) so that it is clear they work. I tested them in Binder, so it should be smooth. Well, except it has the conda-specific instead of a general name, so perhaps leave it for later. I will move all these files anyway and will have to touch the CI then.

The validation_plots notebook works but says (`# Format dataframe` cell):

```
/srv/conda/envs/notebook/lib/python3.8/site-packages/pandas/core/indexing.py:1675: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  self._setitem_single_column(ilocs[0], value, pi)
```

`paper_example_plots.ipynb` would use a title or intro paragraph. There is also some comment out code - I would just delete that.

**Review by wenzeslaus — APPROVED** [review]
I think this is good to go.

I still think it would be better to have `simulation_use_cases.ipynb` with cells computed since the inputs are not in the repo. Maybe we can merge this and leave that for another PR.

**Comment by wenzeslaus** [comment]
I'm going to merge this and open an issue for including a computed `simulation_use_cases.ipynb`.

**Pull request merged**